contracts/strategies/traderJoe/strategy-joe-avax-fief.sol

```diff
@@ -0,0 +1,151 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+import "../strategy-joe-rush-farm-base.sol";
+
+/// @notice The strategy contract for TraderJoe's AVAX/FIEF Liquidity Pool with JOE and AVAX rewards
+contract StrategyJoeAvaxFief is StrategyJoeRushFarmBase {
+    /// @dev LP and Token addresses
+    uint256 public avax_fief_poolId = 52;
+    address public joe_avax_fief_lp = 0x939D6eD8a0f7FC90436BA6842D7372250a03fA7c;
+    
+    address public fief = 0xeA068Fba19CE95f12d252aD8Cb2939225C4Ea02D;
+
+    constructor(
+        address _governance,
+        address _strategist,
+        address _controller,
+        address _timelock
+    )
+        public
+        StrategyJoeRushFarmBase(
+            avax_fief_poolId,
+            joe_avax_fief_lp,
+            _governance,
+            _strategist,
+            _controller,
+            _timelock
+        )
+    {}
+
+    /// @notice **** State Mutations ****
+    /// @dev Collect token fees and add liquidity to base pair
+    function harvest() public override onlyBenevolent {
+        IMasterChefJoeV2(masterChefJoeV3).deposit(poolId, 0);
+        
+        /// @dev Get balance of native AVAX and wrap AVAX into ERC20 (WAVAX)
+        uint256 _avax = address(this).balance;
+        if (_avax > 0) {                                    
+            WAVAX(wavax).deposit{value: _avax}();
+        }
+        
+        /// @dev Check token balances, take fee for each token, then update balances
```

Comment:
Good to see comments following the Natspec format but the usage of the dev tag here is not in line. The dev tag should be used on declarations of functions, variables, events, contracts etc. Inside the function itself you don't need to add the tag. 
See docs here for more info:
https://docs.soliditylang.org/en/v0.8.11/natspec-format.html#tags

Reply:
updated